tests/components/test_producer_app.py

```diff
@@ -19,7 +19,7 @@
 
 class TestProducerApp:
     PRODUCER_APP_NAME = "test-producer-app-with-long-name-0123456789abcdefghijklmnop"
-    PRODUCER_APP_CLEAN_NAME = "test-producer-app-with-long-name-0123456789abc-clean"
+    PRODUCER_APP_CLEAN_NAME = "test-producer-app-with-long-n-clean"
```

Comment:
Why did this change?

Reply:
It's trimmed a bit shorter now because of the `${pipeline_name}-` prefix, see below. Does that answer your question?

Reply:
Aha now I understand it 👍